2024-06-24/minutes.md

```diff
@@ -1,3 +1,188 @@
+
+[Pranshu] Awesome! Looking forward to some progress in the next call.
+
+[Ruth] Yeah.
+
+### Is there an objective around closing issues? (@rumyra)
+
+[Ruth] I had a question around closing issues because we have contributors closing lots of issues. I found out the answer earlier, which is there is no objective. People just wanted to come in and close loads of issues. So I guess the things. Mention here is you're gonna see a lot of activity over the next few weeks because some of our contributors are trying to reduce that issue count, which is fine. This is totally fine. Just because they want to. There was no other reason. They just wanted to do it. Which is cool.
+
+### Inline icons [discussion](https://github.com/orgs/mdn/discussions/654#discussioncomment-9746229) (@rumyra)
+[Ruth] The inline icons discussion on code. This is probably something most relevant to you, Onkar. This is the idea of putting experimental and deprecated icons throughout the site so not just in the side bars but making sure they're clear on pages and in our reference pages and things like that.
+
+I want to see the vote on the discussion to see how maintainers feel we this should be done. But I don't think at this time it's a good idea for us to implement any way to do it because of the platform changes that are coming. So we have go through and have like, including macros and doing all of that kind of stuff and in like 3 months that'll all be mute and null because we'll be doing it in a completely different way.
+ 
+[Florian] Yeah, I can also add on that's 1 of the things I realized that required inconsistent. So when I read it the side bars. Actually, I'm adding more of the where we previously did not. At them and I need to figure out if you wanna don't wanna have them.
+
+[Ruth] Yeah, exactly.
+
+[Florian] And, and at the same time it's it's quite easy like there's a global post processing of all links where we have this context so just injecting them is like really nothing to do.
+
+[Ruth] That's what I'm trying to say on. Like I, I want people to vote. But no implementation to happen at this time, that's what I'm saying.
+
+[Florian] I was wondering about one case I ran into and that I was not aware of like there's guide pages that have like a feature status is that
+
+[Ruth] Yes, because they've got a BCD key. And we're using the BCD key for the baseline status.
+
+[Florian] But the feature is it called feature status, don't get me wrong.
+
+[Ruth] Okay. So yeah, I'm interested to see how maintainer feel about it. I think we also need to ask our users, but we can roll that into some decisions that we're gonna be making about the platform anyway. That's it for me.
```

Comment:
```suggestion
[Ruth] Yeah, that's what I'm trying to say. I want people to vote. But no implementation will happen at this time.

[Florian] I was wondering about one case I ran into and that I was not aware of — some guide pages have a feature status

[Ruth] Yes, because they've got a BCD key. We're using the BCD key for the baseline status.
```